src/main/java/hudson/plugins/sonar/SonarRunnerBuilder.java

```diff
@@ -453,6 +454,9 @@ protected Object readResolve() {
   @Extension
   public static final class DescriptorImpl extends BuildStepDescriptor<Builder> {
 
+    public static final boolean beforeV2 = JenkinsRouter.BEFORE_V2;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS00115)